util/headerreader/header_reader.go

```diff
@@ -217,35 +229,40 @@ func (s *HeaderReader) broadcastLoop(ctx context.Context) {
 					if errors.Is(err, rpc.ErrNotificationsUnsupported) {
 						pollOnlyOverride = true
 					} else if time.Now().After(nextSubscribeErr) {
+						s.setError(errors.Wrap(err, "failed subscribing to header"))
```

Comment:
Agreed. We'd used this errors package in classic because `fmt.Errorf` wrapping didn't exist yet I think, but now that it does we should stop using it.

Reply:
To clarify @joshuacolvin0 the new scheme is `fmt.Errorf("failed subscribing to header: %w", err)`

Reply:
:+1: